### ISSUE TITLE
What version of Coq should non-CI minimization default to?

### ISSUE THREAD

**Issue by JasonGross** [coq/bot#183]
Currently it's "latest released", and there's no good way to configure it when invoking the minimizer.  Perhaps it should be configurable?

**Comment by JasonGross** [comment]
Maybe something like coqbot minimize optional-coq-version?  This would help with https://github.com/coq/coq/issues/15325#issuecomment-991105413

**Comment by proux01** [comment]
Indeed, note that for coq/coq#15325 I already did the minimization by end, so no hurry.

**Comment by erikmd** [comment]
> Perhaps it should be configurable?

I guess it would be a nice feature indeed!

> Maybe something like coqbot minimize optional-coq-version?

yes… in this case, I guess one could have `optional-coq-version` = [any available tag from the `coqorg` registry](https://hub.docker.com/r/coqorg/coq#supported-tags) ?
e.g.
```
coqbot minimize dev

or

coqbot minimize 8.14-native-ocaml-4.07
```
